exporter/awskinesisexporter/internal/translate/translate_jaeger.go

```diff
@@ -15,10 +15,10 @@
 package translate
 
 import (
+	jaegertranslator "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger"
```

Comment:
Be sure to run `goimports` on these files:

```
goimports -local github.com/open-telemetry/opentelemetry-collector-contrib -w <filename>
```

Reply:
Yes, this was fixed in the latest commit 🎉